testinfra/utils/ansible_runner.py

```diff
@@ -16,8 +16,9 @@
 import ipaddress
 import json
 import os
+import re
 import tempfile
-from typing import Any, Callable, Iterator, Optional, Union
+from typing import Any, Callable, Iterator, List, Optional, Set, Union
```

Comment:
This and the suggestion below get this working on lower versions of python, probably replacing the desire for back porting to 8.2
```suggestion
from typing import Any, Callable, Dict, Iterator, List, Optional, Set, Union
```